src/Api/appsettings.Development.json.example

```diff
@@ -1,16 +0,0 @@
-﻿{
```

Comment:
Maybe this happened inadvertently, but you should left the example file on the project.